docs/bundledplugins/pluginmanager.rst

```diff
@@ -94,15 +94,15 @@ octoprint.plugin.pluginmanager.reconnect_hooks
 
 .. py:function:: reconnect_hooks_hook(*args, **kwargs)
 
-   Returns additional hooks defined by the plugin for which the plugin manager
-   should display the "You should reconnect to your printer" message on plugin
-   install/uninstall/enabling/disabling.
+Returns additional hooks defined by the plugin for which the plugin manager
```

Comment:
Doesn't this de-indentation change how that is rendered? 🤔

Reply:
Actually it doesn't. I tested this to be sure. With the indentation, the Python code-block below was not rendering correctly. I don't know why, maybe because there was double indentation and then there was more indentation in the Python code. I decided to remove the indentation for the entire section and that worked just fine.